### ISSUE TITLE
logo is too close to title

### ISSUE THREAD

**Issue by JeffreySarnoff** [JuliaCon/JuliaConSubmission.jl#5]
The logo smashes into (vertically) longer titles, add a little vertical space after the logo.

**Comment by matbesancon** [comment]
Yeah I didn't manage to make this fit right, if you want to PR this I'd gladly take it
